src/sh-tabs.js

```diff
@@ -121,7 +122,7 @@ class ShTabs extends React.Component {
     }
 }
 
-ShTabs.propTypes = {
+ShTabs.PropTypes = {
```

Comment:
propTypes should be camel case starting with a lower case letter: https://facebook.github.io/react/docs/reusable-components.html